### PR TITLE
Add clarification footnote to exact search k-NN blog post

### PR DESCRIPTION
**Pull request by ryanbogan**
### Description
This PR updates a recent blog post for boosting exact search k-NN performance.  A footnote is added to specify the recommended settings for users to achieve the best performance and be able to replicate the benchmarking results
 
### Issues Resolved
N/A

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-11-13-boosting-k-nn-exact-search.md [hunk 1]
```diff
@@ -110,3 +110,8 @@ The tests showed that OpenSearch's new SIMD support and optimized memory access
 ## What's next for exact k-NN search?
 
 Future OpenSearch versions will provide even more k-NN search flexibility. You'll be able to switch between exact and approximate search at query time. Additionally, future versions will provide the ability to specify which fields build indexes for exact and approximate search types. Stay tuned for these updates as we continue to improve OpenSearch's k-NN search capabilities.
+
+
+#### Note:
```
Comment (kolchfa-aws) [inline _posts/2024-11-13-boosting-k-nn-exact-search.md:115]:
```suggestion
#### Note
```

File: _posts/2024-11-13-boosting-k-nn-exact-search.md [hunk 2]
```diff
@@ -110,3 +110,8 @@ The tests showed that OpenSearch's new SIMD support and optimized memory access
...
+
+It is required to enable the setting `index.knn: true` to use the optimized Lucene format. With this setting in OpenSearch 2.17, the approximate nearest neighbors data structures were also built.  In OpenSearch 2.18, we introduced a new setting `index.knn.advanced.approximate_threshold`.  If you are only performing exact search, set this to  either `-1` or `Integer.MAX - 2` in order to speed up indexing time.
```
Comment (kolchfa-aws) [inline _posts/2024-11-13-boosting-k-nn-exact-search.md:117]:
```suggestion
To use the optimized Lucene format in OpenSearch 2.17, set `index.knn` to `true` in order to build approximate nearest neighbor data structures.  In OpenSearch 2.18, a new `index.knn.advanced.approximate_threshold` setting is available. If you are performing only exact searches, set this value to `-1` or `Integer.MAX - 2` to reduce indexing time.
```

**Review by kolchfa-aws — APPROVED** [review]

**Comment by ryanbogan** [comment]
I'm actually going to push one more change quickly to just have the setting be `-1` so people don't get confused if they don't have a coding background

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Pull request merged**